### PR TITLE
Feature sidebar

### LINKED ISSUE #15: Add sidebar
Screenshot from 2019-10-23 16-23-57

### PR DESCRIPTION
**Pull request by DannyFeliz**
Fix #15

### CODE REVIEW

**Comment by DannyFeliz** [comment]
Closing this because the Netlify integration was missing

File: src/components/home/NavBar/index.jsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import React from "react";
 import logo from "assets/images/logo.svg";
+import hamburger from "assets/images/hamburger-icon.svg";
```
Comment (leninpaulino) [inline src/components/home/NavBar/index.jsx]:
```suggestion
import hamburgerIcon from "assets/images/hamburger-icon.svg";
```

File: src/components/home/NavBar/index.jsx [hunk 2]
```diff
@@ -42,6 +43,9 @@ const NavBar = () => {
           </li>
         </ul>
         <button className="login-button">{getAccountButton()}</button>
+        <button onClick={handleToggleSidebar} className="hamberguer-button">
+          <img src={hamburger} alt="Hamburger menu icon" />
```
Comment (leninpaulino) [inline src/components/home/NavBar/index.jsx]:
```suggestion
          <img src={hamburgerIcon} alt="Hamburger menu icon" />
```

File: src/components/home/Sidebar/styles.scss [hunk 3]
```diff
@@ -0,0 +1,30 @@
+.sidebar__container {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  z-index: 999999;
+  width: 300px;
+  background: #3b4144;
```
Comment (leninpaulino) [inline src/components/home/Sidebar/styles.scss]: I think we should start using variables for all the colors in the app.

**Pull request closed without merging**